chapters/background.tex

```diff
@@ -1,4 +1,13 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+Since Viola and Jones proposed "Rapid Object Detection using a Boosted Cascade of Simple Features" in 2001, their concept is the basis of most face detection approaches.
+It's an extremely fast and solid machine learning approach for object detection in videos and pictures, like e. g. faces.
+
+In detail the concept is distinguished by three key components.
+The first is the “Integral Image”, a new image representation which allows the detector to compute features very fast.
+Secondly an AdaBoost based learning algorithm selects a small number of promising visual features and constructs very efficient classifiers.
+The last component is a method which combines increasingly more complex classifiers in a “cascade” where most of the computation time is spend on critical object-like parts of the given image by discarding non-promising regions early in the analysis process \cite{viola_jones}.
```

Comment:
I think you can omit the "more" in "increasingly more".

"most of the computation time is **spent**"

Reply:
Thank's. I'm sure 'increasingly more' is suitable here